frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx

```diff
@@ -27,7 +28,7 @@ const setup = () =>
   )
 
 describe('MetricsStep', () => {
-  it('should render Crews and RealDone components', () => {
+  it('should render Crews components', () => {
```

Comment:
It seems that not only crews components in the document

Reply:
Got that, will double check and fix that later.